docs/source/en/api/pipelines/kandinsky.mdx

```diff
@@ -112,6 +117,29 @@ One cheeseburger monster coming up! Enjoy!
 
 ![img](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/kandinsky-docs/cheeseburger.png)
 
+<Tip warning={true}>
```

Comment:
Instead of adding it as a warning could we add a new section about 2.2? Think the models are too different to be treated as a warning